src/executor/drop_executor.cpp

```diff
@@ -10,6 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <include/catalog/index_catalog.h>
```

Comment:
Use formatter.

Reply:
Actually, you have to manually fix this. It should be:

`#include "catalog/index_catalog.h"`

And it should come after "executor/drop_executor.h".